src/components/SearchCard.tsx

```diff
@@ -61,18 +60,13 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
       () => facility.offers.map((o) => Number(o.price.public)),
       [facility]
     );
-    const handleSelect = useCallback(() => onSelect(facility.id), []);
 
     if (facility.offers.length < 1) {
       return null;
     }
 
     return (
-      <Card
-        ref={ref}
-        onMouseOver={handleSelect}
```

Comment:
this handle select supposed to highlight pin of the chosen facility on map. I believe we must keep it

Reply:
Shouldn't the desired effect be `onMouseClick`? This `onMouseOver` handler was from a previously dropped feature I think

Reply:
nope, onMouseClick should open facility page. Its been mentioned that pins does not highlight now, so it could be caused by removing parts of this logic(may be some time earlier). Is this  `onMouseOver` required to remove? I believe its important tot keep it

Reply:
The logic for selecting a facility marker is actually in the `Mapbox` component and the logic to select a facility is still there. This was used earlier when hovering to allow the card displace to the right. This effect is no longer in use